extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/job/Job.java

```diff
@@ -1,20 +1,26 @@
 /**
- * Copyright (c) 2014-2017 by the respective copyright holders.
+ * Copyright (c) 2017 by the respective copyright holders.
```

Comment:
According to [1] - The range in license header denotes the start year of the content creation and the current year.

[1] - https://eclipse.org/legal/copyrightandlicensenotice.php